week_1/2_apr/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,100 @@
+.header {
+    height: 55px;
+    display: flex;
```

Comment:
@codal-kcolaco Can you please check all the possibles and what come with that?

Reply:
@abhi-codal Yes